charts/argo-cd/values.yaml

```diff
@@ -595,6 +595,14 @@ server:
   ## Annotations to be added to ArgoCD rbac ConfigMap
   rbacConfigAnnotations: {}
 
+  # Name of the configmap used for the RBAC config. Must be set to `argocd-rbac-cm` as the values are hardcoded in
+  # ArgoCD. See https://github.com/argoproj/argo-cd/blob/master/common/common.go
+  rbacConfigMapName: "argocd-rbac-cm"
```

Comment:
I’m not sure we should offer the possibility to configure the ConfigMap name if it’s hardcoded on Argo CD side. It’s error prone in my opinion.

Reply:
That makes sense. Will remove